app/services/recaptcha_validator.rb

```diff
@@ -1,7 +1,7 @@
 class RecaptchaValidator
   VERIFICATION_ENDPOINT = 'https://www.google.com/recaptcha/api/siteverify'.freeze
 
-  class ValidationError < StandardError; end
```

Comment:
This removal isn't strictly relevant, but I noticed while I was here. This is unused, and leftover from some earlier iterations of the original implementation.